execute_feasibility_test.py

```diff
@@ -0,0 +1,318 @@
+    new_history_result = {"date": now}
+    new_history_result["result"] = new_result
+
+    is_new_date = True
+    for i in range(len(history_report["reports"])):
+        if history_report["reports"][i]["date"] == now:
+            history_report["reports"][i] = new_history_result
+            is_new_date = False
+            break
+    if is_new_date:
+        history_report["reports"].append(new_history_result)
+
+    with open(HISTORY_REPORT_FILE, "w") as f:
+        json.dump(history_report, f)
+
+
+def send_table_to_conf(table, conf_user, conf_pw, conf_api_base_url, conf_page_id, page_title, page_content):
+    table_html = table.get_html_string()
+    page_content = f'{page_content}{table_html}<br></br>'
+
+    base64_bytes = base64.b64encode(f'{conf_user}:{conf_pw}'.encode('ascii'))
+    base64_message = base64_bytes.decode('ascii')
+    header = {
+        'Authorization': f'Basic {base64_message}',
+        'Content-Type': 'application/json'
+    }
+
+    res = requests.get(f'{conf_api_base_url}/content/{conf_page_id}',
+                       headers=header)
+
+    version_number = res.json()['version']['number']
+    content_update = {
+        'title': page_title,
+        'type': 'page',
+        'body': {'storage': {'value': page_content,
+                             'representation': 'storage'
+                             },
+                 },
+        'version': {'number': version_number + 1}
+    }
+    res = requests.put(f'{conf_api_base_url}/content/{conf_page_id}',
+                       headers=header, json=content_update)
+
+
+def send_query_and_get_results(query, backend_base_url, backend_client_id, client_secret, keycloak_token_url, wait_result_secs_feas):
+    access_token = connect_to_keycloak(backend_client_id, client_secret, keycloak_token_url)
+    print(f'Sending query: {query}')
+    query_id = send_test_query_and_get_id(access_token, query, backend_base_url)
+    print(f'Query ID: {query_id}')
+    print(f'Sleep for {wait_result_secs_feas} seconds to wait for results')
+    sys.stdout.flush()
+    time.sleep(int(wait_result_secs_feas))
+    access_token = connect_to_keycloak(backend_client_id, client_secret, keycloak_token_url)
+    return get_results(query_id, access_token, backend_base_url)
+
+
+def execute_history_query(backend_base_url, backend_client_id, client_secret, keycloak_token_url, wait_result_secs_feas, conf_user, conf_pw, 
+                          confluence_api_base_url, confluence_page_id_hist, send_results_confluence):
+    load_constants()
+    history_report = load_history_report()
+    history_query = load_json_file(HISTORY_QUERY_FILE)
+    results = send_query_and_get_results(history_query, backend_base_url, backend_client_id, client_secret, keycloak_token_url, wait_result_secs_feas)
+    converted_result = convert_results(results)
+
+    update_and_save_history_report(history_report, converted_result)
+
+    history_table = convert_to_table(history_report["reports"], "date") 
```

Comment:
@bastianschaffer the length of entries to include in the report to be sent to confluence should be configurable via env var - otherwise the table on confluence will eventually get too large to be useful.
-> Default should be 14 entries